### PR TITLE
GovStack sandbox documentation. SND-236-Create-documentation-proposal-about-Sandbox-in-GovStack-GitBook-space

### PR DESCRIPTION
**Pull request by oleksiidn**
@rachellawson could you take a look GovStack sandbox documentation?

### CODE REVIEW

File: spec/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# GovStack Sandbox
+
+The GovStack Sandbox is a unified environment for all the GovStack and digital government
+ecosystem actors to discover, understand, engage, experiment, innovate, and build digital
+government solutions using the [GovStack platform](http://govstack.gitbook.io/).
```
Comment (rachellawson) [inline spec/README.md]: We probably want to link to the main website here, https://govstack.global

File: spec/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+all the systems an application interfaces with. This especially enables government agencies and digital government 
+players to experiment with innovative GovTech products or services within a well-defined space and duration. The design 
+of the GovStack sandbox encapsulates the business logic and data necessary to represent multiple GovStack capabilities 
+such as APIs, building blocks, use cases and transaction flows. This helps developers and digital government agencies 
+to achieve a more holistic view of the GovStack.
+## Get started
+
+The sandbox is reusable and replicable, which means you can set up your own GovStack sandbox within your own environment.
+
+To get more details check out the [infrastructure documentation](https://github.com/GovStackWorkingGroup/sandbox-infra).
```
Comment (rachellawson) [inline spec/README.md]: Presumably, we only want one place to document the sandbox? Is it going to be this repo or the other?

File: spec/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+to achieve a more holistic view of the GovStack.
+## Get started
+
+The sandbox is reusable and replicable, which means you can set up your own GovStack sandbox within your own environment.
+
+To get more details check out the [infrastructure documentation](https://github.com/GovStackWorkingGroup/sandbox-infra). 
+
+## Candidates for building blocks
+In the Sandbox, you have the freedom to configure your own setup and run any building blocks of your choice.
+List of available BB candidates:
```
Comment (rachellawson) [inline spec/README.md]: Is this more “building blocks we have already built, which may be of help to re-use or guide your configuration of your own building block”

Comment (rachellawson) [reply spec/README.md]: Also, how will we ensure this list stays current and how can someone see a list of candidates built?

**Review by rachellawson — COMMENTED** [review]
A few comments that might help get this first version ready

**Comment by rachellawson** [comment]
Oh, and add a Jira issue number to the title of the PR!!

File: spec/README.md (hunk 3 above)
Comment (oleksiidn) [reply spec/README.md]: "List of available BB candidates" is a misleading sentence.
 We currently have no official building blocks.
Here are the candidates for test purposes.

I have updated this part.

File: spec/README.md (hunk 2 above)
Comment (oleksiidn) [reply spec/README.md]: This particular repository is the main documentation for Sandbox.
The [infrastructure repository](https://github.com/GovStackWorkingGroup/sandbox-infra) is the part of the sandbox that is responsible for setting up the environment for the sandbox instance.

I updated this part too.

Comment (oleksiidn) [reply spec/README.md]: The sandbox team is currently focused on testing GovStack's theory of running BB candidates in a single environment. As a result, the team provides vendor-independent configuration.
Anyone else can take this configuration and check it out for themselves.
Perhaps in the future a public playground will be built.
This is one of the reasons why we have many repositories.

**Review by rachellawson — APPROVED** [review]
I think it looks great. Merge and then I’ll set up in GitBook

**Pull request merged**